hostinfo/hostinfo.go

```diff
@@ -20,6 +20,7 @@ import (
 	"tailscale.com/tailcfg"
 	"tailscale.com/util/dnsname"
 	"tailscale.com/util/lineread"
+	"tailscale.com/util/winutil"
```

Comment:
at this point I'd rather we move this packageType code to `hostinfo_windows.go` which already exists.

Otherwise Linux builds end up depending on a `winutil` package which is harmless (it's a stub on Linux) but looks weird.

But mostly because once we're talking about registry values, having the code in a portable file seems misplaced, even if winutil makes it possible.